### PR TITLE
Improve kubelet cgroup

### PR DESCRIPTION
**Pull request by yanxuean**
**What this PR does / why we need it**:
1.Use arg cgroupRoot,not nodeConfig.CgroupRoot
    Using both arg cgroupRoot and nodeConfig.CgroupRoot is confused in function NewQOSContainerManager
2.improve cgroupmanager in qosContainerManager
3. improve arg "cgroupRoot" type in NewQOSContainerManager

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/kubelet/cm/qos_container_manager_linux.go [hunk 1]
```diff
@@ -95,8 +95,8 @@ func (m *qosContainerManagerImpl) Start(getNodeAllocatable func() v1.ResourceLis
 
 	// Create containers for both qos classes
 	for qosClass, containerName := range qosClasses {
-		// get the container's absolute name
-		absoluteContainerName := CgroupName(containerName)
+		// get the container's abstract name
```
Comment (resouer) [inline pkg/kubelet/cm/qos_container_manager_linux.go]: Why change this name? This var is mean to "absolute path of cgroup name"

Comment (yanxuean) [reply pkg/kubelet/cm/qos_container_manager_linux.go]: It is only a conversion to CgroupName. I think it maybe typo. Do you think?

```
// CgroupName is the abstract name of a cgroup prior to any driver specific conversion.
type CgroupName string
```

Comment (resouer) [reply pkg/kubelet/cm/qos_container_manager_linux.go]: I'm afraid not, it is literally the absolute path of cgroup, please leave it.

Comment (yanxuean) [reply pkg/kubelet/cm/qos_container_manager_linux.go]: OK. Tks

**Comment by yanxuean** [comment]
@vishh    PTAL , Thanks.

**Comment by yanxuean** [comment]
ping for approve   @Random-Liu

**Comment by yanxuean** [comment]
ping for approve

**Comment by yanxuean** [comment]
@yujuhong   ping for approve .  thanks.

**Comment by yanxuean** [comment]
@yujuhong ping for approve . thanks.

**Comment by yanxuean** [comment]
@vishh   thanks ，Can you give me approve-no-issue?

**Comment by yanxuean** [comment]
Thanks @yujuhong

**Pull request merged**